<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Text_widget</class>
 <widget class="QDialog" name="Text_widget">
  <property name="windowTitle">
   <string>Send command</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btn_add">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_rem">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_edt">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_up">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_down">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cb_append"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_run">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_rem</tabstop>
  <tabstop>btn_edt</tabstop>
  <tabstop>btn_up</tabstop>
  <tabstop>btn_down</tabstop>
  <tabstop>cb_append</tabstop>
  <tabstop>btn_run</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
